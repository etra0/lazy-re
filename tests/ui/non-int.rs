
use lazy_re::lazy_re;

#[repr(C, packed)]
#[lazy_re]
struct Foo {
    #[lazy_re(offset = 'a')]
    a: usize,
}

fn main() {}
