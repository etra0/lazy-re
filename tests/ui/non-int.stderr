error: expected integer literal
 --> tests/ui/non-int.rs:7:24
  |
7 |     #[lazy_re(offset = 'a')]
  |                        ^^^
